jobs/birthdayOfTheDay.ts

```diff
@@ -0,0 +1,40 @@
+import { TextChannel } from "discord.js"
+import moment from "moment";
+
+async function getBdayUsers(date) 
+{
+  const users = await prisma.user.findMany({
+    where: {
+        birthday: {
+          birthday: date
+        }
+    }
+  })
+  
+  const arrayIds = []
+  
+  for (let i = 0; i < users.length; i++)
+    arrayIds.push(users[i].discordId)
+
+  return arrayIds
+}
+
+const birthdayOfTheDay = {
+  cronPattern: '0 12 * * *', // every day at 12:00
+  channel: '💬｜general',
+  async execute(channel: TextChannel) {
+    const momentVariable = moment(new Date, 'MM/DD/YYYY')
+    var dateFormatted = momentVariable.format('MM/DD')
+    const date = new Date(dateFormatted)
+
+    const arrayIds = (await getBdayUsers(date)).slice();
```

Comment:
After you return just users from `getBdayUsers`, it will be an array that could be empty or contain the users so you can just start a loop over that:
```js
const users = await getBdayUsers(date)
for (const user of users) {
  ... // just access id to send like <@${user.discordId}>
}